backend/metamodel/syson-sysml-metamodel/src/test/java/org/eclipse/syson/sysml/impl/NamespaceImplTest.java

```diff
@@ -496,6 +511,51 @@ public void resolveVisibleTest() {
 
     }
 
+    @DisplayName("Test resolve")
+    @Test
+    public void resolveTest() {
+        var testModel = new TestModel();
+
+
```

Comment:
Why there are multi empty lines here? Please remove them and check every other files.

Reply:
Done